src/MainWindow.cpp

```diff
@@ -694,6 +694,34 @@ void MainWindow::deleteRecord()
     }
 }
 
+void MainWindow::selectCurrentTabTableLines(int firstLine, int lastLine)
```

Comment:
Part of the code in this function could be refactored with the old code by passing the table model and table widgets as parameters. This could be made after merging, though. It is not indispensable.

Reply:
I though doing that would be nice but the method selectTableLine was used in a lot of places so I left it as is.

Reply:
Ok, you have my approval :+1: